src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -0,0 +1,61 @@
+/**
+ * Special-cases for the sphinx-design lib
+ * Mainly to make it compatible with the multime themes supported by pydata-sphinx-theme
```

Comment:
what does multime mean? can you use a different word?

Reply:
I wanted to write "multiple" that's a typo. solved in https://github.com/pydata/pydata-sphinx-theme/pull/641/commits/ed2924beafde746353d5a2cc6d18e4ad6cab22fa